### DISCUSSION TITLE
direct-direct for reflectance

### DISCUSSION THREAD

**Discussion by robertweitl** [LBNL-ETA/pyradiance#44] [Q&A]
What do you think of how to get the reflectance values of the BSDF. 
Is there a method/way to define the outgoing direction "t", "r"?
with pr.BSDF('bsdf.xml') as sd:
    sval = sd.eval(30, 45, 145, 90)

**Comment by taoning** [comment]
These are the available tag for direct_hemi
t: hemispherical transmittance 
ts: specular transmittance
td: diffuse transmittance
r: hemispherical reflectance
rs: specular reflectance
rd: diffuse reflectance
s: scattering

sd.eval would be useful for sampling many individual outgoing rays to evaluate the trans/reflec for a specific outgoing solid angles.
